### PR TITLE
[Task #xxxxx]: Xử lý logic cho trang search

### PR DESCRIPTION
**Pull request by ngmthaq**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/xxxx)

## WHAT this PR does?

- Xử lý logic cho trang search

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Review by quangdm-2954 — APPROVED** [review]

**Comment by ngmthaq** [comment]
ready

File: resources/js/user/app.js [hunk 1]
```diff
@@ -1,28 +1,48 @@
...
-    ajax.main.authorpage(this.getAttribute("data-id"));
-});
+
 $(document).on("click", ".menu-item", function (event) {
     ajax.main.playlistPage(this.getAttribute("data-id"));
 });
+
+$(document).on('submit', ajax.search.formEl, function (e) {
+  e.preventDefault();
+  ajax.search.search();
```
Comment (manhtd-1083) [inline resources/js/user/app.js:39]:
```suggestion
    e.preventDefault();
    ajax.search.search();
```
indent 4 spaces, sửa tương tự các chỗ khác

**Comment by ngmthaq** [comment]
ready

**Comment by ngmthaq** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**